build.sbt

```diff
@@ -6,7 +6,7 @@ lazy val V = new {
   def munit = "0.7.29"
 }
 val scala212 = "2.12.15"
-val scala213 = "2.13.6"
+val scala213 = "2.13.8"
 val scala3 = "3.0.2"
```

Comment:
This will need `3.1.x` for Native

Reply:
(and will force downstream to upgrade)

Reply:
Seems we also need fansi released :thinking: